src/main/java/com/java_essential/kopaiev/essential/homework/lesson1/task2/Main.java

```diff
@@ -0,0 +1,40 @@
+package com.java_essential.kopaiev.essential.homework.lesson1.task2;
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        rectangleCalculation();
+    }
+
+    public static void rectangleCalculation()
+    {
```

Comment:
Better to start class, constructors and methods body on the same line as the name.

Reply:
fixed